### PR TITLE
Add checks for remote branches -> issue #2

### PR DESCRIPTION
**Pull request by willbenica**
- fail if local branch isn't known to remote
- use force option to override this behaviour
- add force to the config.yaml

### CODE REVIEW

File: delete_old_branches/delete_old_branches.py [hunk 1]
```diff
@@ -79,6 +79,28 @@ def get_local_branches(git_repo):
     return branches
 
 
+def get_remote_branches(git_repo):
```
Comment (mschaaf) [inline delete_old_branches/delete_old_branches.py]: better name that fits the description or vice versa

File: delete_old_branches/delete_old_branches.py [hunk 2]
```diff
@@ -239,7 +268,12 @@ def main():
         )
 
     local_branches = get_local_branches(path)
-    branches_to_delete = generate_to_delete_list(local_branches, do_not_delete)
+    remote_branches = get_remote_branches(path)
```
Comment (mschaaf) [inline delete_old_branches/delete_old_branches.py]: move into the else case to avoid unnecessary execution

File: delete_old_branches/delete_old_branches.py [hunk 3]
```diff
@@ -239,7 +268,12 @@ def main():
...
+    if force:
+        branches_to_delete = generate_to_delete_list(local_branches, do_not_delete)
+    else:
+        branches_in_remotes = [branch for branch in local_branches if branch in remote_branches]
```
Comment (mschaaf) [inline delete_old_branches/delete_old_branches.py]: 1. better name: branches_that_exists_local_and_remote
2.a move to method with nice name what it does or
2.b branch -> branch_that_exists_local_and_remote

File: delete_old_branches/delete_old_branches.py [hunk 4]
```diff
@@ -239,7 +268,12 @@ def main():
...
+        branches_to_delete = generate_to_delete_list(branches_in_remotes, do_not_delete)
```
Comment (mschaaf) [inline delete_old_branches/delete_old_branches.py]: line 273 a line 276 could be merged to one line outside of the if/else short before line 277

File: dm_tools/generate_entities.py [hunk 5]
```diff
@@ -184,6 +185,9 @@ def main(dm_server):
         push_wbk_to_dm(file, dm_server)
         clean_up_json(file)
 
+    # Shut down logging handlers
+    logging.shutdown()
+
```
Comment (mschaaf) [inline dm_tools/generate_entities.py]: unrelated change!

**Comment by mschaaf** [comment]
Anyways thanks for fixing

File: delete_old_branches/delete_old_branches.py (hunk 1 above)
Comment (willbenica) [reply delete_old_branches/delete_old_branches.py]: Fixed the description.

Comment (willbenica) [reply delete_old_branches/delete_old_branches.py]: Matched the description to what this function does
